subiquity/server/server.py

```diff
@@ -387,6 +433,21 @@ def report_finish_event(self, context, description, status):
             listener.report_finish_event(context, description, status)
         self._maybe_push_to_journal("finish", context, description)
 
+    def report_info_event(self, context: Context, message: str) -> None:
+        for listener in self.event_listeners:
+            listener.report_info_event(context, message)
```

Comment:
So this an similar changes the interface that something passed to add_event_listener is expected to implement. I get that this is a bit theoretical given that there is only ever likely to be one type passed to that function but maybe we should annotate that somehow?

Reply:
I suppose we should make an abstract `EventListener` class? I can implement this

Reply:
Yeah it's probably worth it for the documentation value. A bit makeworky but not very much?